packages/cli/src/workflow-execute-additional-data.ts

```diff
@@ -302,6 +302,9 @@ function hookFunctionsPush(): IWorkflowExecuteHooks {
 						retryOf: this.retryOf,
 						workflowId,
 						workflowName,
+						flattedRunData: data?.resultData.runData
```

Comment:
I don't think `data` can be undefined here. If the typing suggest that it can, then we should fix the typing, or add type assertions.

Reply:
let's fix in another PR.